src/expandable-block/block.scss

```diff
@@ -167,4 +167,23 @@
       margin-left: $iui-s;
     }
   }
+
+  &.iui-borderless {
+    .iui-header,
+    .iui-expandable-content {
+      background-color: transparent;
```

Comment:
This results in the background color being `iui-color-background-3` from `iui-expandable-block` instead of `iui-color-background-2` from `iui-body`.

We can either change the background color here or make `iui-expandable-block-borderless` its own class and set its background color to `transparent` similar to how `iui-button-borderless` is written

Reply:
Changing it here is quite simple, so you could just do that. All you'd need to add is `&`.
![image](https://user-images.githubusercontent.com/9084735/152829503-34d70044-7ff2-4a6c-b74f-f559c5643ebf.png)

If you were to create a `iui-expandable-block-borderless`, would it be _in addition to_ `iui-expandable-block` or _instead_ of it? The former wouldn't change much as you would still have to do some overrides. And the latter would require some extra work to make sure all styles are included.

In the case of buttons, the reason `iui-button` doesn't have anything is just because we have the `iui-default` modifier for applying default colors. So `iui-borderless` and `iui-default` are never used together.

Reply:
Your suggested change makes the most sense

Change has been implemented

Reply:
Dont forget to update test images

Reply:
The test images have been updated

Reply:
Yes, it will pass because the diff checker has issues with recognizing color changes. But if you approved those images before changing this background color, then the images are "wrong" even if the tests pass.

To force an update, you can manually delete the images, then run the tests (causing failures), and approve the new images.